tests/tests/test_rvs/test_dynrvs.py

```diff
@@ -0,0 +1,33 @@
+import phoebe
+import numpy as np
+
+
+def test_dynrvs(verbose=False):
+    """
+    This test checks whether PHOEBE computes dynamical RVs without building
+    a mesh (as otherwise run_compute would fail with atm out-of-bounds errors), 
+    and whether the computed RVs are close to the flux-weighted RVs.
+    """
+    b = phoebe.default_binary()
+    b.add_dataset('rv', compute_times=phoebe.linspace(0, 1, 21), dataset='dynrv')
+    b.set_value_all('rv_method', 'dynamical')
+    b.set_value_all('teff', 1000000)
+    b.run_compute()
+
+    b.set_value_all('teff', 5772)
+    b.add_dataset('rv', compute_times=phoebe.linspace(0, 1, 21), dataset='fluxrv')
+    b.set_value('rv_method', component='primary', dataset='fluxrv', value='flux-weighted')
+    b.set_value('rv_method', component='secondary', dataset='fluxrv', value='flux-weighted')
+    b.run_compute(eclipse_method='only_horizon')
+
+    if verbose:
+        print('maximum primary RV offset:', np.abs(b['value@rvs@primary@dynrv']-b['value@rvs@primary@fluxrv']).max())
+        print('maximum secondary RV offset:', np.abs(b['value@rvs@secondary@dynrv']-b['value@rvs@secondary@fluxrv']).max())
+
+    assert np.allclose(b['value@rvs@primary@dynrv'], b['value@rvs@primary@fluxrv'], atol=0.3)
+    assert np.allclose(b['value@rvs@secondary@dynrv'], b['value@rvs@secondary@fluxrv'], atol=0.3)
```

Comment:
these will run faster if you're explicit and always provide the tags (`set_value(qualifier='teff', value=10000)` and avoiding twigs).

Reply:
certainly, but this is on the user-end and this code seems more readable to me, no? Either way, I have no strong feelings either way.

Reply:
We can always clean it up later - but do think it makes sense to try to make tests run efficiently so that CI doesn't take long (but we're talking 0.1 seconds here or something like that, so not a huge deal).